collector-exoplayer/src/main/java/com/bitmovin/analytics/exoplayer/ExoPlayerAdapter.java

```diff
@@ -203,58 +212,66 @@ public void onLoadingChanged(boolean isLoading) {
 
     @Override
     public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
-        long videoTime = getPosition();
-        Log.d(TAG, String.format("onPlayerStateChanged: %b, %s", playWhenReady, ExoUtil.exoStateToString(playbackState)));
+        try {
```

Comment:
I'm not sure if we need that at all for Exoplayer. Again, @Tigraine , any opinion?

Reply:
I'd add it anyways .. We really want to make sure that the collector never crashes the App 🤔 . So when in doubt: add more try/catch 😆